helpers/helpers.go

```diff
@@ -449,8 +449,11 @@ func ParseCSR(in []byte) (csr *x509.CertificateRequest, rest []byte, err error)
 // It does not check the signature. This is useful for dumping data from a CSR
 // locally.
 func ParseCSRPEM(csrPEM []byte) (*x509.CertificateRequest, error) {
+	var der []byte
 	block, _ := pem.Decode([]byte(csrPEM))
-	der := block.Bytes
```

Comment:
more elegant way is that check block and throw error immediately, instead of relying on CSR parsing to catch the empty input.